specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-10-01/scheduledActions.json

```diff
@@ -0,0 +1,940 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CostManagementClient",
+    "version": "2022-10-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.CostManagement/scheduledActions": {
+      "get": {
```

Comment:
With this linter error,

`
Operations API must be implemented for '/providers/Microsoft.CostManagement/operations'.
Location: Microsoft.CostManagement/stable/2022-10-01/scheduledActions.json#L35`

Looks like you need to declare the provideres/Microsoft.CostManagement/operations
or "operations metadata api" per RPC

Reply:
Most of the response definitions you need for that are also in common types BTW

Reply:
I am planning to do that in a separate PR through another json file, costManagement.json

Reply:
I will probably move that to common-types.json

Reply:
Our Operations API is also returning an Id property in the response, hence couldn't use the common Operation definition.

Reply:
Did you find you can use 'allOf' with your own definition to inherit any parts of the base definitions?

Reply:
Yes, I ended up using allOf and referencing the base definition.